src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -63,32 +63,36 @@ const NavHeader = (props) => {
 
   const AnimatedBlurView = Animated.createAnimatedComponent(BlurView);
 
-  const headerHeight = isHeaderHide
-    ? dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT2
-    : dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT;
+  const additionalHeaderHeight =
+    isHeaderHide || initialData?.coverImage !== undefined || coverPath !== null
+      ? dimen.normalizeDimen(12)
+      : 0;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for calculating `additionalHeaderHeight` seems to be a bit confusing. It's not clear why the presence of a cover image or the header being hidden would affect the additional height. Consider adding comments to explain this logic, or refactor it for clarity if possible.



<!-- This is an auto-generated comment by OSS CodeRabbit -->